### DISCUSSION TITLE
Is that possible to use current version of maqs with latest Selenium 4.6.0

### DISCUSSION THREAD

**Discussion by krthknan** [CognizantOpenSource/maqs-dotnet#206] [General]
We are using maqs-dotnet and now we implemented Selenium 4.6.0. We would like to know, is that possible to work with Selenium 4.6.0 which have inbuild WebDriverManager. 
Are you working on any update with Selenium 4.6.0, if yes when can we expect?

**Comment by DanMcNamee** [comment]
We are using Selenium 4.7.0 and .Net 6 with MAQS 9.0.3 without issue.
